docker/Dockerfile

```diff
@@ -46,6 +46,8 @@ RUN chmod +x $YARN
 # Stage 2 - Install dependencies
 FROM skeleton AS deps
 
+COPY $EXTERNAL_SOURCE_NESTED/dynamic-plugins/ ./dynamic-plugins/
+#RUN find dynamic-plugins \! -name "package.json" -mindepth 2 -maxdepth 2 -print | xargs rm -rf
```

Comment:
if this is a useful cleanup step, could move this down to line 102 after

```
RUN $YARN install --frozen-lockfile --production --network-timeout 600000
```

Reply:
I think it's not useful in fact.